pegasus/sites.v3/code.org/views/course_table.haml

```diff
@@ -69,70 +70,136 @@
+  - courses << { name: "App Lab", starts: 9, ends: 12 }
+  - courses << { name: "Text compression widget", starts: 9, ends: 12 }
+  - courses << { name: "Pixelation widget", starts: 9, ends: 12 }
+  - courses << { name: "Cryptography widget", starts: 9, ends: 12 }
+  - courses << { name: "Internet simulator", starts: 9, ends: 12 }
+
+  - courses << { heading: "Standalone activities" }
+  - courses << { name: "Unplugged", starts: 0, ends: 6, link: "/curriculum/unplugged" }
+  - courses << { name: "Hour of Code", starts: 0, ends: 12, link: "/learn" }
 
 .courses
-  .group
-    .grouprow.bar-6
+  .group{style: "overflow: hidden"}
+    .grouprow.bar-6{style: "opacity: 0.7"}
       Elementary school
-    .grouprow.bar-3
+    .grouprow.bar-3{style: "opacity: 0.85"}
       Middle school
-    .grouprow.bar-4
+    .grouprow.bar-4{style: "opacity: 1"}
       High school
   .grade.break
-    .graderow.bar-1
+    .graderow.bar-1{style: "opacity: 0.7"}
       K
     - (1..12).each do |n|
-      .graderow.bar-1
+      - opacity = n.between?(0,5) ? 0.7 : n.between?(6,8) ? 0.85 : 1
+      .graderow.bar-1{style: "opacity: #{opacity}"}
         = n
-    - courses.each_with_index do |course, course_index|
-      .course.break{id: "course-#{course_index}", onclick: "$('#courseextra-#{course_index}').slideToggle(); $('.extra').not('#courseextra-#{course_index}').slideUp();"}
-        - unless course[:starts] == 0
-          %div{class: "bar-#{course[:starts]}"}
-            &nbsp;
-        %div{class: "courserow bar-#{course[:ends] - course[:starts] + 1}", style: "opacity: #{course[:opacity] || 1}"}
-          .title
-            =course[:name]
-          - if course[:group]
-            .groupname
-              =(course[:group])
-          .extra{id: "courseextra-#{course_index}", style: "display: none"}
-            - if course[:subtitle]
-              .subtitle
-                =course[:subtitle]
-                %br/
-            .description
-              =course[:description]
-            - if course[:link]
-              %a{href: course[:link] }
-                Learn more
-
-  .notes{style: "clear: both"}
+  .coursebodycontainer{ style: "position: relative; overflow: hidden; clear: both"}
+    .coursebodyback{ style: "position: absolute; top: 0; left: 0; height: 100%; width: 100%; z-index: -100"}
+      - (1..13).each do |n|
+        - opacity = n.between?(0,6) ? 0.6 : n.between?(7,9) ? 0.8 : 1
+        .graderow.bar-1{style: "height: 100%; background-color: rgba(200, 200, 200, #{opacity})"}
+    .coursebodymain{style: "width: 100%"}
+      - courses.each_with_index do |course, course_index|
+        - if course[:heading]
+          %h2.course.break
+            = course[:heading]
+        - else
+          %a{href: course[:link] || "" }
+            .course.break{id: "course-#{course_index}", xonclick: "$('#courseextra-#{course_index}').slideToggle(); $('.extra').not('#courseextra-#{course_index}').slideUp();"}
```

Comment:
(Brendan and I talked offline) - `xonlick` here is WIP hack and we'll remove or fix later